content/posts/Devash_Recommendation_System_Final_Report.md

```diff
@@ -0,0 +1,141 @@
+---
+title: 'Project Recommendation System: Final Report (GSoC 2021- Devansh Dixit)'
+date: 2021-08-13T21:32:37+05:30
+draft: false
+author: Devansh Dixit
+type: post
+---
+
+# Recommendation System
+
+The recommendation system is the first ML project for [CircuitVerse](https://github.com/CircuitVerse/CircuitVerse). The basic idea is to recommend similar circuits (which are also popular) to the users to increase user engagement and reach of good projects. In this report, I have explained every decision or technology in simple words to give a better understanding and to give an idea of why we did, what we did.
```

Comment:
Just say this project aims to add the functionality of a  recommendation system in first line.